cpp/src/bitmask/valid_ops.cu

```diff
@@ -167,10 +167,13 @@ void count_valid_bits(valid32_t const * const masks32,
  * @Returns  GDF_SUCCESS upon successful completion 
  *
  * ----------------------------------------------------------------------------*/
-gdf_error gdf_count_nonzero_mask(gdf_valid_type const * masks, int num_rows, int * count)
+gdf_error count_nonzero_mask(gdf_valid_type const * masks,
+                                 int num_rows,
+                                 int& count,
```

Comment:
Output parameters passed by reference are potentially confusing at the call site. Much clearer to pass output parameters by pointer.